charts/rancher-monitoring/100.1.1+up19.0.3/templates/rancher-monitoring/dashboards/rancher/performance-dashboards.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.rancherServiceMetrics.enabled }}
+{{- $selector := (include "rancher.serviceMonitor.selector" .) -}}
```

Comment:
calls template to identify if a valid selector can be made, if nil and rancher is not detected to come up with the default, this outputs nothing. Therefore, the if statement below evaluates as false and nothing is deployed.